### DISCUSSION TITLE
extract only neccesary code from components to save tickets when sending request

### DISCUSSION THREAD

**Discussion by RunningCircles42** [Erikborgis/react-automatic-testing#15] [General]

**Comment by Erikborgis** [comment]
Remove comments and those things??

Many things are important like imports and stuff so that it knows what type of testing library it should use, if its for React or React-Native makes a big difference.

I think this would be super hard to single out the "important" or "not important" code from a component as we want to make tests for the whole component.

**Reply by RunningCircles42** [reply]
I was thinking more about removing \n or like empty lines with no code but perhaps it does not count it as tickets

**Reply by Erikborgis** [reply]
I think that tickets is only words? We will have to read about the api to see

**Comment by RunningCircles42** [comment]
I have looked closer on it and its only words, tecken and numbers and since the cost is extremely low anyway it would be unnecessary work. I therfore close this discussion/issue
